ajax/libs/billboard.js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "billboard.js",
+  "description": "Re-usable easy interface JavaScript chart library, based on D3 v4+",
+  "author": "NAVER Corp.",
+  "filename": "billboard.min.js",
+  "homepage": "http://naver.github.io/billboard.js/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/naver/billboard.js"
+  },
+  "keywords": [
+    "d3",
+    "chart",
+    "graph",
+    "svg"
+  ],
+  "license": "MIT",
+  "npmName": "billboard.js",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/!(*.+(map))"
```

Comment:
I don't think we need to exclude the `map` files. Seems fine to use `**/*` inside `dist`.